features/util.py

```diff
@@ -16,8 +17,11 @@
     "container": {"series": "image.release", "machine_type": "image.type"},
 }
 SLOW_CMDS = ["do-release-upgrade"]  # Commands which will emit dots on travis
-SOURCE_PR_TGZ = os.path.join(tempfile.gettempdir(), "pr_source.tar.gz")
-SOURCE_PR_UNTAR_DIR = os.path.join(tempfile.gettempdir(), "behave-ua")
+UA_TMP_DIR = os.path.join(tempfile.gettempdir(), "uaclient")
```

Comment:
I think we should only rename this to indicate that is coming from the integration tests, like `uaclient-behave`

Reply:
Good idea!